docs/eventing/samples/kafka/channel/README.md

```diff
@@ -0,0 +1,144 @@
+# Apache Kafka CRD default channel
+
+You can install and configure the Apache Kafka CRD (`KafkaChannel`) as the default channel configuration in Knative Eventing. 
+
+## Prerequisites
+
+You must ensure that you meet the [prerequisites listed in the Apache Kafka overview](../README.md).
+
+You must also have the following tools installed:
+- `curl`
+- `sed`
+
+## Creating a `KafkaChannel` channel CRD
+
+Install the `KafkaChannel` sub-component on your Knative Eventing cluster:
+   ```
+   curl -L "https://github.com/knative/eventing-contrib/releases/download/v0.8.2/kafka-channel.yaml" \
+    | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.kafka:9092/' \
+    | kubectl apply --filename -
+   ```
+
+> Note: The above assumes that you have Apache Kafka installed in the `kafka`, as discussed [here](../README.md)!
+
+Once the `KafkaChannel` API is available, create a new object by configuring the YAML file as follows:
+
+```
+cat <<-EOF | kubectl apply -f -
+---
+apiVersion: messaging.knative.dev/v1alpha1
+kind: KafkaChannel
+metadata:
+  name: my-kafka-channel
+spec:
+  numPartitions: 1
+  replicationFactor: 3
+EOF
+```
+
+You can now set the `KafkaChannel` CRD as the default channel configuration.
+
+## Specifying the default channel configuration
+
+To configure the usage of the `KafkaChannel` CRD as the [default channel configuration](channels/default-channels.md), edit the `default-ch-webhook` ConfigMap as follows:
+
+```
```

Comment:
Can this be applied in the same way? 
ie.
```
cat <<-EOF | kubectl apply -f -
---
apiVersion: v1 ...
 ...
EOF
```

Reply:
Yes